src/Adapter/Presenter/Cart/CartPresenter.php

```diff
@@ -95,6 +95,10 @@ private function includeTaxes()
      */
     private function presentProduct(array $rawProduct)
     {
+        $assembler = new \ProductAssembler(Context::getContext());
```

Comment:
Please set ProductAssembler in use parts.

Reply:
And it's :ok_hand: for me.

Reply:
Done! 👍